package.json

```diff
@@ -1005,6 +1018,41 @@
           "type": "string",
           "default": "texdoc",
           "markdownDescription": "Define the location of texdoc executable.\nThis command is used to show a package documentation."
+        },
+        "latex-workshop.progress.runIconType": {
+          "type": "string",
+          "default": "Parenthesised",
+          "enum": [
+            "Parenthesised",
+            "Circled",
+            "Solid Circled",
+            "Full Stop"
+          ],
+          "enumDescriptions": [
+            "⑴ ⑵ ⑶ …",
+            "① ② ③ …",
+            "❶ ❷ ❸ …",
+            "⒈ ⒉ ⒊ …"
+          ],
+          "markdownDescription": "The style of number to use to inidcate the run number"
+        },
+        "latex-workshop.progress.barLength": {
+          "type": "integer",
+          "default": 15,
+          "markdownDescription": "How many characters long the progress bar should be"
+        },
+        "latex-workshop.progress.barStyle": {
```

Comment:
The `barStyle` is missing a default value. Without that there is no progress information for users who don't set the bar style. This is especially problematic since the vscode settings ui pretends the first value (`Block Width`) is set but when you open the json settings editor you see there is actually no value specified (should be improved / fixed by Microsoft/vscode#58724).

Reply:
Good catch! Fixed.
Also, any thoughts on https://github.com/James-Yu/LaTeX-Workshop/pull/1157#issuecomment-460202074 ?

Reply:
I'm not quite sure what you are suggesting in that comment, but I'm also not sure the values for the preamble are wrong. This is how my output looks when making a change in the middle of the document (notice that the time for the preamble is only short in the first run and that every run changes different pages):
![image](https://user-images.githubusercontent.com/8282547/52204377-44607400-2874-11e9-9692-15d22d89777b.png)

I think it may well be that that is just how latex works / the nature of the output produced by pdflatex.